initialise/initialise.go

```diff
@@ -51,9 +51,17 @@ func (e *ExternalServiceList) GetConsumer(ctx context.Context, cfg *config.Confi
 		Offset:       &kafkaOffset,
 		KafkaVersion: &cfg.KafkaVersion,
 	}
+	if cfg.KafkaSecProtocol == "TLS" {
```

Comment:
Consider creating a constant for `"TLS"` - The constant will reduce the chance of mistakes but also define what the expected value should be, what case it should be in etc.

Reply:
That's a good idea. Done